daemon/execdriver/native/create.go

```diff
@@ -91,6 +91,8 @@ func generateIfaceName() (string, error) {
 func (d *driver) createNetwork(container *configs.Config, c *execdriver.Command) error {
 	if c.Network.HostNetworking {
 		container.Namespaces.Remove(configs.NEWNET)
+		container.Namespaces.Remove(configs.NEWUTS)
```

Comment:
@LK4D4 I don't follow.  I had to set Hostname="" because libcontainer itself would blow up saying I can't specify a hostname if UTS isn't private.  So I set Hostname to ""